index.html

```diff
@@ -27,15 +44,712 @@
       </nav>
+
+      <div id="featured-blocks" class="grid-container">
+        <div id="main-block" class="block bg-img-1">
+        <img id="cover-logo" class="hide-on-mobile" alt="The Next Web" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            <div class="date">
+              <span>Monday — July 6, 2020</span>
+            </div>
+            <div class="heading">
+              <h1>
+                New AI project captures Jane Austen’s thoughts on social media
+              </h1>
+              <span class="author">by <a href="#">Thomas Macaulay</a></span>
+              <span class="time">
+                <i class="far fa-clock"></i>
+                37 minutes ago</a></span>
+              <span class="shares">
+                <i class="fas fa-share-alt"></i>
+                2</a></span>
+            </div>
+        </div>
+        <div id="nested-featured-blocks">
+          <div class="block bg-img-2">
+            <div class="heading">
+              <h1>
+               A comprehensive guide on how to detect faces with Python
+              </h1>
+              <span class="author">by <a href="#">Live Code Stream</a></span>
+              <span class="time">
+                <i class="far fa-clock"></i>
+                1 hour ago</a></span>
+              <span class="shares">
+                                <i class="fas fa-share-alt"></i>
+                0</a></span>
+                
+            </div>
+            </div>
+          <div class="block bg-img-3">
+            <div class="heading">
+              <h1>
+              Everything we know about the OnePlus Nord
+              </h1>
+              <span class="author">by <a href="#">Ivan Mehta</a></span>
+              <span class="time">
+                <i class="far fa-clock"></i>
+                1 hour ago</a></span>
+              <span class="shares">
+                                <i class="fas fa-share-alt"></i>
+
+                1</a></span>
+            </div>
+            </div>
+          </div>
+        </div>
+      </div>
     </header>
+    <main>
+      <h1 class="orange-heading"><a href="#">Latest news</a> <i class="fas fa-chevron-right"></i></h1>
+      <div class="latest-news grid-container article-container">
+       
+        <article class="article-item flex-container">
+          <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/Untitled-design-95-516x270.png"
+            alt="New AI project captures Jane Austen's thoughts on social media" />
+         <div>
+            <h3> <a href="#">New AI project captures Jane Austen's thoughts on social media</a></h3>
+            <div class="article-meta">
+              <span class="author"><a href="#">Thomas Macaulay</a></span>
+              <span>.</span>
+            
+              <span class="time">3 hours ago</span>
+            </div>
+         </div>
+        </article><article class="article-item flex-container">
```

Comment:
## Html semantics
Wrap your articles or any section ith `<section>` tag

Reply:
`<section>` tags give an error in the HTML validator because they should have an `<h1>`  as a direct child.